types/aws-lambda/trigger/appsync-resolver.d.ts

```diff
@@ -12,7 +12,7 @@ export type AppSyncBatchResolverHandler<TArguments, TResult, TSource = Record<st
 >;
 
 // See https://docs.aws.amazon.com/appsync/latest/devguide/security-authz.html#aws-lambda-authorization
-export type AppSyncAuthorizerHander<TResolverContext = undefined> = Handler<
+export type AppSyncAuthorizerHandler<TResolverContext = undefined> = Handler<
```

Comment:
I'm definitely for this fix but I'm concerned this counts as a breaking change.
Sadly I'm not too sure what the official expected approach might be in DefinitelyTyped for a simple typo as opposed to radically changed types.

My view is it's unlikely to affect many (is npm likely to grab a patch release sporadically so some developers end up on one vs the other?). An alternative option might be to re-export the fixed name and deprecate the old one?
*Other maintainers - thoughts? Happy to approve as-is otherwise!*

Reply:
This would be a breaking change and will need to be adjusted to be backwards compatible before we can release. 

I think your idea about re-exporting the fixed and deprecating the old. That way, at least it doesn't break people's code but they get warned about the deprecation.